tests/cli/test_install_notebooks.py

```diff
@@ -0,0 +1,10 @@
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+from dsgrid.utils.run_command import check_run_command
+
+
+def test_install_notebooks():
+    with TemporaryDirectory() as tmpdir:
+        check_run_command(f"dsgrid install-notebooks --path={tmpdir}")
+        assert list((Path(tmpdir) / "dsgrid-notebooks").iterdir())
```

Comment:
Should we also check expected length of list and/or at least one of the filenames?

Reply:
Since it is a fresh directory, checking for non-zero length should be good enough. I didn't include a name because I was expecting the existing name to change, causing someone to have to fix the test. But, that's ok. I'll add a name check.